solutions/numpy_core_tests/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#FROM python:3.10-rc-slim
```

Comment:
Prebuilt python:3.9-slim image also works.

```c
FROM python:3.9-slim

RUN python3 -m pip install --upgrade pip
RUN python3 -m pip install numpy pytest hypothesis 

RUN mkdir /app
WORKDIR /app
ADD app.py /app

RUN rm /bin/sh && ln -s /bin/bash /bin/sh

ENV PYTHONUNBUFFERED=1

CMD ["/bin/bash"]
```

Reply:
Used `python:3.9-slim` as the base image instead of building cpython from source.